src/util/InlineEditor.js

```diff
@@ -1,9 +1,8 @@
 import React, {Component} from 'react';
 import AceEditor from 'react-ace'
-import 'brace/mode/scala'
 
-import FlixMode from './FlixMode'
-import 'brace/theme/chrome';
+import 'ace-builds/src-noconflict/mode-flix'
+import 'ace-builds/src-noconflict/theme-chrome'
 
```

Comment:
Can this entire class be dropped?

Reply:
I believe so. It's strictly about highlighting, right?

Reply:
Yeah.

Reply:
Can do in follow up PR

Reply:
If we're both talking about `FlixMode` then it's already removed :)

Reply:
I meant InlineEditor?

Reply:
Or maybe it is needed as a separate component?

Reply:
Ah, right now `InlineEditor` is used for the uneditable code boxes

Reply:
OK